contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -791,7 +791,7 @@ read/modify/write cycle, by verifying that the current value of resourceVersion
 matches the specified value.
 
 The resourceVersion is currently backed by [etcd's
-modifiedIndex](https://coreos.com/etcd/docs/latest/v2/api.html).
+mod_revision](https://etcd.io/docs/latest/learning/api/#key-value-pair).
```

Comment:
@porridge  is this mainly due to migration from etcdv2 to etcd v3 ?

Reply:
It's mostly due to the old URL disappearing
![Screenshot from 2022-11-04 08-41-57](https://user-images.githubusercontent.com/489420/199919355-36d86d9f-959f-42ed-a895-d04dd23346de.png)